lib/pharos/phases/configure_etcd_changes.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+require 'json'
+
+module Pharos
+  module Phases
+    class ConfigureEtcdChanges < Pharos::Phase
+      title 'Configure etcd member changes'
+
+      def call
+        store_initial_cluster_state
+
+        return if initial_cluster_state != 'existing' || !etcd.healthy?
```

Comment:
This can't be used to repair an unhealthy etcd cluster by adding a new member or replacing failed members?

Reply:
Currently no, I have not tested recovery from unhealthy state.

Reply:
Right now the "recover from broken etcd cluster" case isn't working... but that might be because we have a `GET /v2/keys/foo` healthcheck on the etcd pod, so the kubelet ends up killing all the etcd members once the cluster goes unhealthy...

Once the `etcd` restarts, it seems like there's some kind of startup delay where it just times out all client requests if the cluster is unhealthy cluster... but I did eventually seem to get a response for `GET /v2/members` before the kubelet killed the etcd pod again, so I think it *might* be possible to recover an unhealthy cluster by replacing the failed member?

Reply:
I tried this but it seems that broken cluster peer does not respond at all. I think it's out of scope of this PR to fix this broken etcd cluster.

Reply:
Still thinking this might behave better without the `etcd.healthy?` check... then it would fail right on the `Configure etcd member changes` => get member list, rather than continuing on to the `Configure etcd` and then just hanging because the new member can't join the cluster.